tests/tasks/spacy/__init__.py

```diff
@@ -0,0 +1 @@
+import pytest
```

Comment:
To pass the vanilla prefect tests do:
```
import pytest

pytest.importorskip("spacy")
```

Reply:
thanks! I had been searching everywhere for a missing dependency